### ISSUE TITLE
508-defect-3: Links that open in a new tab should warn users ahead of time

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#45653]
## Point of contact

Josh Kim

## Severity level

3, Moderate. Should be fixed in 1-3 sprints post-launch.

## Details

Several issues exist when forcing a link to open in a new tab or window:
- Screen reader users, screen magnifier users, and users with certain cognitive impairments can become disoriented when they are taken to a new window
- The use of the back button, the most used button within the browser, is broken.
- Certain user agents, like kiosks, are unable to open new windows or tabs.

From WCAG guidance:

> “In general, it is better not to open new windows and tabs since they can be disorienting for people, especially people who have difficulty perceiving visual content.”
[G200: Opening new windows and tabs from a link only when necessary](https://www.w3.org/TR/WCAG20-TECHS/G200.html)

For more, [view our guidance on links (google slides)](https://docs.google.com/presentation/d/14AwUoFOg_Ix1Txl9oePtPD1ntOpylBWwBJMYq9f5vmo/edit?usp=sharing)

<img width="1499" alt="links" src="https://user-images.githubusercontent.com/14154792/184416236-c98697d0-067f-4fbc-b01a-06969ff52a95.png">
<img width="1541" alt="links-1" src="https://user-images.githubusercontent.com/14154792/184416239-81b66f5c-5b31-4ad1-8496-59f177c48970.png">
<img width="1530" alt="links2" src="https://user-images.githubusercontent.com/14154792/184416241-abdf70db-feb6-4d29-a0cf-baa4e52100d2.png">

## Reproduction steps

1. [Sign in as user 299 and go to step 2 of 2 letter list](https://staging.va.gov/records/download-va-letters/letters/letter-list)
2. Click on any of the links, confirm they all open new tabs without warning

## Proposed solution or next steps

We can either:
- Don't open the links in a new tab (which may be undesired if users will be signed out and can't directly return back to it)
  - Note: if we transition to @skylerschain 's new design where this is all stored on a static page (as opposed to part of a form flow), we should no longer have a need to open in a new tab as users can easily return to this page with the back button
- Provide a warning that links will open in a new tab by appending either:
  - a text warning (opens in new tab) e.g. `Download the latest version of Adobe Acrobat Reader (opens in new tab)`
  - an external link icon with alt text that states the prior

**Comment by jacobworrell** [comment]
We can't think of a good reason why we should be opening a new tab for these links. 

In addition, we think we should update the "download the latest version" link text to be more specific about what they're downloading.

**Comment by sortizsh** [comment]
We should wordsmith the paragraph text "if you're still having trouble opening..." to indicate it will open in a new tab, and to make the user know specifically what they're downloading and why. USDS guidelines indicate to only to open a new tab if a users loses progress or data.

recommendation: Download the latest Adobe Acrobat (opens new tab). @skylerschain  can you look at this TY
